function-support.html

```diff
@@ -1848,7 +1939,8 @@ <h2 id="relations">Relations</h2>
 	<table>
 		<tr>
 			<td>Direct Input:</td>
-			<td style="font-family: KaTeX_Main, KaTeX_AMS;">= &lt; &gt; : ∈ ∋ ∝ ∼ ∽ ≂ ≃ ≅ ≈ ≊ ≍ ≎ ≏ ≐ ≑ ≒ ≓ ≖ ≗ ≜ ≡ ≤ ≥ ≦ ≧ ≫ ≬ ≳ ≷ ≺ ≻ ≼ ≽ ≾ ≿ ⊂ ⊃ ⊆ ⊇ ⊏ ⊐ ⊑ ⊒ ⊢ ⊣ ⊩ ⊪ ⊸ ⋈ ⋍ ⋐ ⋑ ⋔ ⋙ ⋛ ⋞ ⋟ ⌢ ⌣ ⩾ ⪆ ⪌ ⪕ ⪖ ⪯ ⪰ ⪷ ⪸ ⫅ ⫆</td>
+			<td><span style="font-family: KaTeX_Main, KaTeX_AMS;">= &lt; &gt; : ∈ ∋ ∝ ∼ ∽ ≂ ≃ ≅ ≈ ≊ ≍ ≎ ≏ ≐ ≑ ≒ ≓ ≖ ≗ ≜ ≡ ≤ ≥ ≦ ≧ ≫ ≬ ≳ ≷ ≺ ≻ ≼ ≽ ≾ ≿ ⊂ ⊃ ⊆ ⊇ ⊏ ⊐ ⊑ ⊒ ⊢ ⊣ ⊩ ⊪ ⊸ ⋈ ⋍ ⋐ ⋑ ⋔ ⋙ ⋛ ⋞ ⋟ ⌢ ⌣ ⩾ ⪆ ⪌ ⪕ ⪖ ⪯ ⪰ ⪷ ⪸ ⫅ ⫆
+			</span>≔ ≕ ⩴</td>

```

Comment:
Shouldn't these be in the `<span>`?

Reply:
Similarly, `ℂ ℍ ℕ ℙ ℚ ℝ ℤ` above seems to be outside the span, which causes them to render in the wrong font.

Reply:
I disagree.

I expect that some authors will copy-and-paste directly from the function support page into their text editor. So when I say that characters are available for "Direct Input", I need to supply characters at the correct Unicode code point. Otherwise, someone will copy what they think is a double-struck C, but will be a normal C when it lands in their text editor.

So I want to keep these characters as is. I'm sorry about the font, but I want to minimize author confusion.

Reply:
Ah, I see.  That makes sense.  Sorry for missing the subtlety here (especially if it's been discussed before).